README.md

```diff
@@ -94,156 +94,146 @@ Example Job
+```python 
+response = client.profile.parsing.get(source_key="source_key", key="profile_key")
+```
+- Retrieve parsing information using reference
 ```python
-resp = client.profile.parsing.get(source_key="source_key", key="profile_key")
+response = client.profile.parsing.get(source_key="source_key", reference="my_resume")
+```
+- The output should should look like this
+```
+{'code': 200,
+ 'message': 'Profile parsing',
+ ...}
 ```
-* Profile indexing:
-In order to add Json profile you can index them using HrFlow search engine
+### 💾 **Index a Profile in a Source**
+In order to add a Json profile you can index it using HrFlow search engine
+- Index a profile in a source using a JSON file `profile_json`
 ```python
-profile_json = {
-  "source_key": "source_key",
-  
-  "consent_algorithmic": {
-            "owner": {
-                "parsing": True,
-                "revealing": False,
-                "embedding": True,
-                "searching": False,
-                "scoring": True,
-                "reasoning": False
-            },
-            "controller": {
-                "parsing": True,
-                "revealing": False,
-                "embedding": True,
-                "searching": False,
-                "scoring": True,
-                "reasoning": False
-            }
-        },
-  "info" : {
-      "full_name":"Harry Potter",
-      "first_name": "Harry",
-      "last_name": "Potter",
-      "email":"harry.potter@gmail.com",
-      "phone":"0202",
-      "gender": None,
-      "urls": {
-          "from_resume": [],
-          "linkedin":"",
-          "twitter":"",
-          "facebook":"",
-          "github":"",
-          "picture":""},
-      "picture":None,
-  	  "location":{"text": None},
-  	  "summary": "Brief summary"
-  },
-  "text": "test text",
-  "experiences": [{
-      "date_start": "15/02/1900",
-      "date_end": "",
-      "title": "Lead",
-      "company": "Mathematic Departement",
-      "location": {"text":"Paris"},
-      "description": "Developping."
-      }],
-  "experiences_duration":5,
-  "educations": [{
-      "date_start": "12540",
-      "date_end": "12550",
-      "title": "Mathematicien",
-      "school": "University",
-      "description": "Description",
-      "location": {"text":"Scotland"}
-  }],
-  "educations_duration":4,
-  "skills": [{"name":"manual skill", "value": None}, {"name":"Creative spirit", "value": None},
-             {"name":"Writing skills", "value": None}, {"name":"Communication", "value": None}],
-  "languages" : [{"name":"english", "value": None}],
-  "interests": [{"name":"football", "value": None}],
-  "tags":[],
-  "metadatas":[],
-  "labels":[{"stage":"yes", "job_id":"job_id"}],
-  "attachments": []
-}
-resp = client.profile.indexing.add_json(source_key="source_key", profile_json=profile_json)
+response = client.profile.indexing.add_json(source_key="source_key", profile_json=profile_json)
+```
+- The output should be of this form:
+```
+{'code': 201,
```

Comment:
Il te manque un exemple de JSON utilise celui là pour montrer facilement un exemple avec lequel on peut tester https://developers.hrflow.ai/reference/the-profile-object#example-of-profile-json

Reply:
J'ai mis un exemple de JSON mais au début avant de commencer à parler d'API. Dans la section Profile Example